lnd-rpc/src/main/scala/org/bitcoins/lnd/rpc/LndRpcClient.scala

```diff
@@ -8,15 +8,16 @@ import lnrpc._
 import org.bitcoins.commons.jsonmodels.lnd._
 import org.bitcoins.core.currency._
 import org.bitcoins.core.number.UInt32
+import org.bitcoins.core.protocol.BitcoinAddress
 import org.bitcoins.core.protocol.ln.LnInvoice
 import org.bitcoins.core.protocol.ln.currency.MilliSatoshis
 import org.bitcoins.core.protocol.ln.node.NodeId
 import org.bitcoins.core.protocol.script.ScriptPubKey
 import org.bitcoins.core.protocol.transaction.{
   TransactionOutPoint,
-  TransactionOutput
+  TransactionOutput,
+  Transaction => Tx
```

Comment:
Why is this done?

Reply:
otherwise we I have to do `transaction.Transaction` everywhere because there is a `lndrpc.Transaction` too